dependency-suppression/cve-suppressed.xml

```diff
@@ -36,4 +36,48 @@
    ]]></notes>
         <cve>CVE-2022-45688</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: pulsar-client-all-2.11.0.jar (shaded: org.apache.avro:avro-protobuf:1.10.2)
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.avro/avro\-protobuf@.*$</packageUrl>
+        <cve>CVE-2021-43045</cve>
```

Comment:
@noorkhan-92 I don't see a `path` here, we seem to be excluding the entire avro-protobuf dependency here?

@alifahadk